source/site-administrators/studio/content-monitoring.rst

```diff
@@ -14,7 +14,7 @@ What is Content Monitoring?
 
 Content Monitoring is a feature that allows you to configure watches and notifications on your site.
 This will provide an easy way to detect unwanted or outdated content.
-Crafter Studio using the Website_editorial blueprint out of the box will run the process that performs monitoring everyday  and send notifications
+Sites created using Crafter Studio's Website Editorial blueprint out of the box will run the process that performs monitoring everyday and send notifications
```

Comment:
I think we can improve this description:
- Mention that the monitoring process always runs in Studio for all existing sites once a day at noon (time zone of the server)
- Mention that all built-in blueprints include a default configuration (not only editorial)